cmd/livepeer/livepeer.go

```diff
@@ -140,6 +141,8 @@ func main() {
 	monitor := flag.Bool("monitor", false, "Set to true to send performance metrics")
 	version := flag.Bool("version", false, "Print out the version")
 	verbosity := flag.String("v", "", "Log verbosity.  {4|5|6}")
+	metadataAmqpUri := flag.String("metadataAmqpUri", "", "URI for AMQP host to send operation metadata")
+	metadataExchange := flag.String("metadataExchange", "lp_golivepeer_metadata", "Name of AMQP exchange to send operation metadata")
```

Comment:
`lp_broadcaster_metadata` might be a little more apt?

Reply:
So my thinking was to use AMQP `topic` exchanges on our favor here. In the current format, this is what we have:
 - Exchange name is `lp_golivepeer_metadata`: `lp_` would be a prefix for us to add to every exchange populated by livepeer services, not to conflict with users' own exchanges when pointing our services to their rabbits. The next segment would be the name of the "service"/code that generates the events, and then the last segment, which feels optional, specifies the kind of data published in there. In this format the webhooks exchange we have would look like: `lp_api_webhooks` (but I don't really intend to change it, just for comparison)
 - The events go under a topic `${nodeType}.${whatever}`. In the case of stream health events, they will be `broadcaster.stream_health.transcode.${manifestId}`

This means that a consumer can bind their queue to the `lp_golivepeer_metadata` exchange and specify in their "subscription" the exact topics of events that they want:
 - Only stream health events from broadcasters: `broadcaster.stream_health.#`
 - All broadcasters events: `broadcaster.#`
 - All stream health events from any node type, if we ever send those from other node types: `*.stream_health.#`
 - Only the stream health events from a specific stream: `*.stream_health.*.${manifestId}`
 - Every event sent by `go-livepeer` code: `#`

That way we can easily start sending events from other node types as well without having to change any of the logic here. Some alternatives would be:
 - Have a default value of `""`  for the exchange name here, and build the default exchange name using the `nodeType` instead, and then probably remove the `nodeType` prefix from the event topics. One would still be able to subscribe to "all node types" events, but they would need to create multiple bindings for their queues, one for each node type.
 - Have a default value of `lp_broadcaster_metadata` here, and only allow initializing the AMQP producer if the node type is a broadcaster, probably failing the command if an AMQP URI is specified but the node is not a broadcaster. If we want to send events from other node types in the future, would be the same setup as above. And would probably make sense to change flag names here as well? `broadcasterMetadata*` maybe

WDYT? Do you prefer any of those?

Reply:
Hmmmm. I guess we might want to use it with Os someday too — perhaps that'd be useful to someone building a scaled public network platform atop orchestrators. Your reasoning makes sense to me, good enough for now :+1: